pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -295,8 +281,7 @@ let
 
       # This is to ensure expansion of $out.
       libExecPath="${libExecPath}"
-      python build/linux/unbundle/replace_gn_files.py \
-        --system-libraries ${toString gnSystemLibraries}
+      python build/linux/unbundle/replace_gn_files.py --system-libraries ${toString gnSystemLibraries}
```

Comment:
Not sure about these style changes, but doesn't matter anyway as it's fine either way (IMO).